DayrSuryan/DSEthiop12.xml

```diff
@@ -60,7 +60,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   </msItem>
 
                   <msItem xml:id="ms_i2">
-                     <locus from="24r" to="37v"/>
+                     <locus from="24r" to="37r"/>
```

Comment:
was 37v wrong? it matches the following 38r of i3

Reply:
it was wrong, 37v is emtpy